### DISCUSSION TITLE
Lingo CGM

### DISCUSSION THREAD

**Discussion by mihirsavadi** [maheini/FreeStyle-Libre-3-patch#61] [General]
Hey is there anyway to get juggluco to become compatible with Lingo CGM. https://www.hellolingo.com/

This is a new CGM from Abott, and is equivalent in hardware to Freestyle Libre 3. However it mandates an iphone only Lingo app, and every app I've tried on my android doesnt work, including the official libre apps as well as the ones not on the app store - Xdrip+ and what not.

Kind of desperate to make this work. Willing to send a fresh example Lingo module to a dev that can ensure compatibility within a reasonable time frame.

**Comment by gui-dos** [comment]


**Reply by mihirsavadi** [reply]
Not helpful dude

**Comment by maheini** [comment]
Gui-dos DiaBLE is the best approach on iOS for sure.
Besides that, Juggluco might work in order to start the sensor on android (guess you tried that already). Other known solutions aren't available yet - at least nothing I know of.

**Reply by mihirsavadi** [reply]
Hey thanks for the reply. Juggluco doesn't scan it either, it sees it as an unrecognizable libre 3.
